src/Symfony/Component/Validator/Constraints/DivisibleByValidator.php

```diff
@@ -23,7 +23,26 @@ class DivisibleByValidator extends AbstractComparisonValidator
      */
     protected function compareValues($value1, $value2)
     {
-        return (float) 0 === fmod($value1, $value2);
+        $value1 = \abs($value1);
+        $value2 = \abs($value2);
+        $epsilon = 0.0000001;
```

Comment:
What about making this configurable?

Reply:
I‘m not sure whether this would actually add anything. 

Might make sense to add something like FloatUtils to capsiluate some of this logic (comparing floats etc)